blog/2025/keycloak-devday-2026-announcement.adoc

```diff
@@ -0,0 +1,89 @@
+= Keycloak DevDay 2026 Announcement and Call-for-Papers
+:date: 2025-08-07
+:modified: 2025-08-07
+:publish: true
+:author: Sebastian Rose
+
+We (Niko and me (Sebastian)) are excited to announce the next edition of Keycloak DevDay!
+DevDay is expanding to a **2-day event** taking place again in Darmstadt, Germany on **March 5th and 6th, 2026**.
+
+The location is about 30 minutes away from Frankfurt/Main Airport by public transport, see https://www.keycloak-day.dev[] for details.
```

Comment:
"The conference takes place again in Darmstadt, about 30 minutes away from Frankfurt/Main..." ?